config/prov/prov_test.yaml

```diff
@@ -130,6 +130,8 @@ k8s_cortx_deploy:
   log_path: "/etc/cortx/log"
   custom_log_path: "/etc/cortx/custom/log"
   s3_max_start_timeout: 240
+  s3_instances_per_node: 3
```

Comment:
Will this resource allocation work with multiple data pods and multiple rgw instance ?

Reply:
this is the default value provided in solution.yaml, so have kept that , for custom values user can pass that in the test of in jenkins job.

Reply:
For 3 and 5 these are working